deploy-manage/deploy/cloud-on-k8s.md

```diff
@@ -1,23 +1,98 @@
+
+ECK should work with all conformant installers as listed in these [FAQs](https://github.com/cncf/k8s-conformance/blob/master/faq.md#what-is-a-distribution-hosted-platform-and-an-installer). Distributions include source patches and so may not work as-is with ECK.
+
+Alpha, beta, and stable API versions follow the same [conventions used by Kubernetes](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning).
+
+Elastic Stack application images for the OpenShift-certified Elasticsearch (ECK) Operator are only available from version 7.10 and later.
+
+Check the full [Elastic support matrix](https://www.elastic.co/support/matrix#matrix_kubernetes) for more information.
+
+% TBD: discuss if these make sense here
+## Learn more about ECK [k8s_learn_more_about_eck]
+
+* [Orchestrate Elasticsearch on Kubernetes](https://www.elastic.co/elasticsearch-kubernetes)
+* [ECK post on the Elastic Blog](https://www.elastic.co/blog/introducing-elastic-cloud-on-kubernetes-the-elasticsearch-operator-and-beyond?elektra=products&storm=sub1)
+* [Getting Started With Elastic Cloud on Kubernetes (ECK)](https://www.youtube.com/watch?v=PIJmlYBIFXM)
+* [Running the Elastic Stack on Kubernetes with ECK](https://www.youtube.com/watch?v=Wf6E3vkvEFM)
+
+% TBD: discuss where to put this "ask for help info"
+## Ask for help [k8s-ask-for-help]
+
+If you are an existing Elastic customer with an active support contract, you can create a case in the [Elastic Support Portal](https://support.elastic.co/). Kindly attach an [ECK diagnostic](/troubleshoot/deployments/cloud-on-k8s/run-eck-diagnostics.md) when opening your case.
+
+Alternatively, or if you do not have a support contract, and if you are unable to find a solution to your problem with the information provided in these documents, ask for help:
+
+* [ECK Discuss forums](https://discuss.elastic.co/c/eck) to ask any question
+* [Github issues](https://github.com/elastic/cloud-on-k8s/issues) for bugs and feature requests
+
+% TBD: decide if this should be ommited also.
+% This was a "redirect only" in the excel
+## Advanced topics [k8s-advanced-topics]
+
+* [*Deploy ECK on OpenShift*](/deploy-manage/deploy/cloud-on-k8s/deploy-eck-on-openshift.md)
+* [*Deploy ECK on GKE Autopilot*](/deploy-manage/deploy/cloud-on-k8s/deploy-eck-on-gke-autopilot.md)
+* [*Create custom images*](/deploy-manage/deploy/cloud-on-k8s/create-custom-images.md)
+* [*Service meshes*](/deploy-manage/deploy/cloud-on-k8s/service-meshes.md)
+* [*Traffic Splitting*](/deploy-manage/deploy/cloud-on-k8s/requests-routing-to-elasticsearch-nodes.md)
+* [*Network policies*](/deploy-manage/deploy/cloud-on-k8s/network-policies.md)
+* [*Webhook namespace selectors*](/deploy-manage/deploy/cloud-on-k8s/webhook-namespace-selectors.md)
+* [*Stack Monitoring*](/deploy-manage/monitor/stack-monitoring/eck-stack-monitoring.md)
+* [*Deploy a FIPS compatible version of ECK*](/deploy-manage/deploy/cloud-on-k8s/deploy-fips-compatible-version-of-eck.md)
```

Comment:
same concern here @shainaraskas : All those items are included in their relevant places... do you think it's ok to keep that `advanced topics` section here in the main overview?

Reply:
Working on it :)